### PR TITLE
[DOCS] add poetry installation instructions for Teams SDK

### PR DESCRIPTION
**Pull request by swheaton**
Docs improvement to add instructions for installing Teams SDK with `poetry` rather than straightforward `pip`.
Looks like this when generated.
screenshot of generated doc

### CODE REVIEW

File: docs/source/teams/installation.rst [hunk 1]
```diff
@@ -72,6 +78,51 @@ That's it! Any operations you perform will be stored in a centralized location
...
+
+You should then see snippets in the ``pyproject.toml`` file like the following:
+
+.. code-block:: toml
+
+    [[tool.poetry.source]]
+    name = "fiftyone-teams"
+    url = "https://pypi.fiftyone.ai/simple/"
+    default = false
+    priority = explicit
```
Comment (findtopher) [inline docs/source/teams/installation.rst:119]: This snippet actually won't work with 1.5 any longer:
```
[[tool.poetry.source]]
name = "fiftyone-teams"
url = "https://pypi.fiftyone.ai/simple/"
priority = "explicit"
```

is the new swagger... (and will throw an error if you use the old syntax) 

might have to differentiate for different versions?  Or just not show them the innards?

**Comment by spacepirate0001** [comment]
`Warning: Found deprecated key 'default' or 'secondary' in pyproject.toml configuration for source fiftyone-teams. Please provide the key 'priority' instead. Accepted values are: 'default', 'primary', 'secondary', 'supplemental', 'explicit'.`

```
[[tool.poetry.source]]
name = "fiftyone-teams"
url = "https://pypi.fiftyone.ai/simple/"
default = false
explicit = true
```

should be 

```
[[tool.poetry.source]]
name = "fiftyone-teams"
url = "https://pypi.fiftyone.ai/simple/"
priority = "explicit"
```

**Comment by swheaton** [comment]
@spacepirate0001 thanks! Sorry, the image in this PR (and that I sent to you) was not up to date with the content of the branch. We decided to just remove those lines since it will look different depending on 1.4 or 1.5+.

Really, the important part is the `poetry source add ... `, since that will edit the pyproject.toml for you.

**Comment by swheaton** [comment]
I think it's useful to have the exact text though, I'm gonna add it back in along with that note about `kaleido`

**Review by lkaneda — APPROVED** [review]
LGTM

**Pull request merged**